gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,627 @@
+
+
+Before you begin: 
+ 
+Confirm that your Kubernetes cluster is available by following the steps in <<check-cluster>>. Review <<k8-cluster-reqs>> if you have not done so already. These prerequisites help avoid deployment errors. 
+
+After confirming that your Kubernetes cluster is ready, complete the following steps:
+
+. Run the Helm command for deploying your Flex Gateway container and connecting it to Anypoint Platform. 
++
+[source,helm]
+----
+helm -n gateway upgrade -i --create-namespace \
+--wait ingress flex-gateway/flex-gateway \ 
+--set gateway.mode=connected \
+--set-file registration.content=registration.yaml
+----
++
+This command creates the `gateway` namespace and `ingress` if they do not exist. The command syntax for the Helm repository and chart names is `<helm-repo-name>/<helm-chart-name>`. 
++
+When successful, the following occurs:
++
+* The command prints a message indicating an upgrade to your `ingress` release:
++
+----
+Release "ingress" has been upgraded. Happy Helming!
+NAME: ingress
+LAST DEPLOYED: Mon Mar 13 22:44:11 2023
+NAMESPACE: gateway
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+----
++
+The REVISION increments by `1` each time you run this command with the same namespace, repository, and chart names. For example, if you run the command a second time, you find a new revision number (`REVISION: 2`). The `LAST DEPLOYED` date reflects the date of that revision.
++
+If you receive an UPGRADE FAILED response, see <<upgrade-failed,troubleshooting>> information about this issue. 
+
+* A new Flex Gateway container is running in your terminal window.
+
+. Return to Runtime Manager: 
+.. Select *&#8592; Flex Gateways* from the side navigation panel of the Kubernetes *Add a Flex Gateway* page. 
++
+image::gateway-quickstart-k8-return-to-gateway-list.png["Return to Flex Gateway list in Runtime Manager"]
+.. Navigate to your registered container in the *Flex Gateways* page that opens:
++
+image::gateway-quickstart-k8-my-gateway-rtm.png["Flex Gateway my-gateway in Runtime Manager"]
++
+You can use the search field in the page to narrow the list of containers.
+
+.. Make sure that the status of your new container is *Connected*. 
++
+If the status is *Disconnected*, it is likely that the cluster configuration is incorrect. This issue can occur, for example, if the `helm -n gateway upgrade` times out before an `ingress` is created or if the cluster does not allow for a `LoadBalancer` resource. Check <<troubleshooting>> for more information. 
+
+
+[[create-api]]
+== Step 5: Publish and Deploy a Simple API to Your Flex Gateway Container
+
+Create and deploy a simple API to your Flex Gateway container in Anypoint API Manager. 
+
+. In Runtime Manager, click the *View APIs* menu option for your container to navigate to the container's *View APIs* page.
```

Comment:
```suggestion
. In Runtime Manager, click the *View APIs* menu option for your Flex Gateway instance to navigate to its *View APIs* page.
```

Reply:
re repetition, I was concerned an editor would say that the reference of "its" might not be 100% clear (even though seems clear to me :) )